source/_patterns/01-elements/textarea/textarea.scss

```diff
@@ -3,14 +3,13 @@
 @use "@db-ui/base/build/scss/variables" as *;
 @use "@db-ui/base/build/scss/helpers/functions" as *;
 @import "textarea.variables";
-@import "../form-elements";
 
 .elm-textarea {
-	@extend %form-element;
+	@extend %form-element !optional;
```

Comment:
do we really need the optional here? I like the idea that the build breaks in case that the relevant styles aren't present, because elsewhere we would get a "valid" build but an unstyled element, especially as we don't have some kind of a fallback, wouldn't we ?

Reply:
done and deleted all references to %form-element. Left comment in _textarea.scss_ for later